pkgs/tools/package-management/nix/default.nix

```diff
@@ -74,7 +75,18 @@ common =
           ''}
         '' else ''
           configureFlagsArray+=(BDW_GC_LIBS="-lgc -lgccpp")
-        '';
+        '') + ''
+          mkdir tmp/
+          substitute corepkgs/config.nix.in tmp/config.nix.in \
+	    --subst-var-by bash ${bash}/bin/bash \
+	    --subst-var-by coreutils ${coreutils} \
+	    --subst-var-by bzip2 ${bzip2}/bin/bzip2 \
+	    --subst-var-by gzip ${gzip}/bin/gzip \
+	    --subst-var-by xz ${xz}/bin/xz \
+	    --subst-var-by tar ${gnutar}/bin/tar \
+	    --subst-var-by tr ${coreutils}/bin/tr
+	  mv tmp/config.nix.in corepkgs/config.nix.in
```

Comment:
Can you indent that properly?

Reply:
To be precise, the `--subst-var-by` lines should be 2 lines more indented than the `substitute` above, and the `mv` on the same level as `substitute` itself.

Reply:
Dammit my editor snuck tabs in

Reply:
It seemed it did that again - can you take another look?